### PR TITLE
Fix detection of Tomcat startup in osg-test

### PR DESCRIPTION
**Pull request by brianhlin**
http://vdt.cs.wisc.edu/tests/20160602-1452/results.html

Test results (look good): http://vdt.cs.wisc.edu/tests/20160602-1452/results.html

### CODE REVIEW

File: osgtest/tests/test_76_tomcat.py [hunk 1]
```diff
@@ -28,6 +27,11 @@ def test_03_deconfig_tomcat_properties(self):
         core.skip_ok_unless_installed(tomcat.pkgname(), 'emi-trustmanager-tomcat')
         files.restore(os.path.join(tomcat.sysconfdir(), 'server.xml'), 'tomcat')
 
+    def test_04_deconfig_catalina_logging(self):
+        core.skip_ok_unless_installed(tomcat.pkgname())
+        if core.el_release() == 5: # limit okskip list
```
Comment (osg-cat) [inline osgtest/tests/test_76_tomcat.py]: I don’t understand this comment in this context…

File: osgtest/tests/test_24_tomcat.py [hunk 2]
```diff
@@ -43,12 +34,39 @@ def test_04_config_tomcat_endorsed_jars(self):
             new_contents = old_contents + "\n" + line
             files.write(tomcat.conffile(), new_contents, owner='tomcat')
 
+    def test_04_configure_gratia(self):
+        core.skip_ok_unless_installed(tomcat.pkgname(), 'gratia-service')
+        command = ('/usr/share/gratia/configure_tomcat',)
+        core.check_system(command, 'Unable to configure Gratia.')
+
     def test_05_start_tomcat(self):
```
Comment (osg-cat) [inline osgtest/tests/test_24_tomcat.py]: Is it absolutely critical to have a Gratia step mingled in with the generic Tomcat steps?

**Comment by osg-cat** [comment]
Overall, I think that this set of changes looks good, but I had one substantive comment that I would like addressed before accepting the whole mess.

File: osgtest/tests/test_24_tomcat.py (hunk 2 above)
Comment (brianhlin) [inline osgtest/tests/test_24_tomcat.py]: Yes: including it in `test_23_gratia.py` results in config changes being overwritten by the earlier tests in `test_24_tomcat.py`.

**Comment by brianhlin** [comment]
I've updated the comment with hopefully a clearer one: https://github.com/opensciencegrid/osg-test/pull/24/commits/a1eeec146034663c93dc8d6e9764c9b9e770476f

I think my comment about the gratia config test in `test_24_tomcat.py` should address your concerns.

**Comment by osg-cat** [comment]
OK, the disentangling of Tomcat and Gratia steps will have to wait until we split apart the test modules more.

**Pull request merged**